marlowe-playground-client/src/Halogen/Monaco.purs

```diff
@@ -223,3 +242,8 @@ handleQuery (SetKeyBindings Vim next) =
     disableVimMode <- liftEffect $ Monaco.enableVimBindings editor
     modify_ (\s -> s { deactivateBindings = disableVimMode })
     pure next
+
+handleQuery (UpdateGetModelMarkers getModelMarkers next) =
+  withEditor \editor -> do
+    void $ H.modify (\s -> s { getModelMarkers = getModelMarkers })
```

Comment:
If you want, I believe `(\s -> s { getModelMarkers = getModelMarkers })` can be replaced with `(_ { getModelMarkers = getModelMarkers })`.

Then _possibly_ with `(_ { getModelMarkers })`.

Reply:
This comment applies in lots of places...

Reply:
apparently you can't do `(_ { getModelMarkers })`